src/middlewared/middlewared/role.py

```diff
@@ -48,27 +48,32 @@ class Role:
     'SYSTEM_AUDIT_WRITE': Role(),
 
     'FULL_ADMIN': Role(full_admin=True, builtin=False),
-    'READONLY': Role(includes=['ALERT_LIST_READ',
-                               'ACCOUNT_READ',
-                               'AUTH_SESSIONS_READ',
-                               'CLOUD_SYNC_READ',
-                               'DATASET_READ',
-                               'ENCLOSURE_READ',
-                               'FAILOVER_READ',
-                               'IPMI_READ',
-                               'KMIP_READ',
-                               'FILESYSTEM_ATTRS_READ',
-                               'NETWORK_GENERAL_READ',
-                               'SHARING_READ',
-                               'KEYCHAIN_CREDENTIAL_READ',
-                               'REPORTING_READ',
-                               'REPLICATION_TASK_CONFIG_READ',
-                               'REPLICATION_TASK_READ',
-                               'SERVICE_READ',
-                               'SNAPSHOT_TASK_READ',
-                               'SUPPORT_READ',
-                               'SYSTEM_AUDIT_READ'],
-                     builtin=False),
+    'READONLY': Role(includes=[
```

Comment:
I wonder if we can do `ROLES['READONLY'] = Role(includes=[role for role in ROLES if role.endswith('_READ')]`)